theseus/geometry/functorch.py

```diff
@@ -33,7 +33,7 @@ def __exit__(self, exc_type: Any, exc_value: Any, traceback: Any) -> None:
         _FunctorchContext.set_context(self.prev)
 
 
-class enable_functorch(object):
+class enable_functorch(_FunctorchContext):
```

Comment:
Is deriving from `_FunctorchContext` intentional since before it was derived from `object`?

Reply:
Correct.